process/transaction/shardProcess.go

```diff
@@ -439,6 +439,8 @@ func (txProc *txProcessor) processMoveBalance(
 		return err
 	}
 
+	// TODO: check if correct that gas is refunded to sender instead of relayer for relayed Txs
+	// TODO: check 2 - create receiptWithReturnedGas even if we are in the senderShard?
```

Comment:
this was taken out in gasPrice-modifier branch - there will be no receipt in case of relayed.

Reply:
this also changed

Reply:
ok